examples/exp_of_DCRNN/signal.py

```diff
@@ -0,0 +1,97 @@
+import torch
+import numpy as np
+from typing import List, Union
+from torch_geometric.data import Data
+
+
+Edge_Index = Union[np.ndarray, None]
+Edge_Weight = Union[np.ndarray, None]
+Features = List[Union[np.ndarray, None]]
+Targets = List[Union[np.ndarray, None]]
+
+
+class StaticGraphTemporalSignal(object):
```

Comment:
Could you please consider the usage of the already implemented [TemporalData](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/data/temporal.py) class? If it is not appropriate, could you please consider turning the `StaticGraphTemporalSignal` class as a subclass of it?